RetrospectiveExtension.Frontend/package.json

```diff
@@ -100,9 +100,11 @@
       "./components/__tests__/setupTests.tsx"
     ],
     "moduleNameMapper": {
-      "office-ui-fabric-react/lib/(.*)$": "office-ui-fabric-react/lib-commonjs/$1"
+      "office-ui-fabric-react/lib/(.*)$": "office-ui-fabric-react/lib-commonjs/$1",
+      "^.+\\.(css|less)$": "<rootDir>/components/__mocks__/cssTestStub.tsx"
     },
     "preset": "ts-jest/presets/js-with-babel",
+    "resetMocks": true,
```

Comment:
reset mock call count, etc. Required for expecting the appropriate number of calls for mocks which are reused across different test files.